src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';').map((pair) => {
```

Comment:
According to the checklist, avoid using loops and use iteration methods instead. The current implementation is correct as it uses 'map', but ensure that no loops are used elsewhere.